docs/src/getting-started/basics.md

```diff
@@ -276,6 +276,35 @@ let contract_id = "0x0123..." // Your contract ID as a string.
 let connected_contract_instance = MyContract::new(contract_id, wallet);
 ```
 
+## Getting the contract call outputs
+
+### Contract call success
+
+If the contract call succeeded, getting the receipts and logs is done this way:
+
+```rust,ignore
+let response = contract_instance.my_method(args).call().await?;
+let logs: Vec<String> = response.logs // This gives out the decoded hex LOGD logs
+let receipts: Vec<Receipt> =  response.receipts // This gives out all the receipts of the transaction
+```
+
+> **Note**: For this to work, because of the `?`, it means the call has to have succeded.
+
+### `ContractCallError`
+
+- If the contract call creates an error, the reason and the receipts of the transaction will be displayed in the Rust `panic`.
+
+- If you want to use the receipts in the case of a `ContractCallError`, you can do:
+
+```rust,ignore
+let contract_call_error = contract_instance.my_error_method(args).call().unwrap_err();
```

Comment:
How do you know it's an error or not before unwrapping? This example isn't complete. All the more reason to have actual examples that are tested.

Reply:
I'm just trying to conform to the instruction `how to read the receipts in case it's an error`. IMO, in case it's an error, the logging done by the `ContractCallError(reason, receipts)` is enough, but I felt like you wanted a code example.

Reply:
Please correct me if I'm wrong, but the case here is if the transaction is valid but reverts, right? If so, then the SDK needs to handle such cases. It can't just panic. A code example is needed. You can keep the example, just expand on it.

Reply:
But we agreed in the issue that the SDK would error on a valid transaction that reverts.

Reply:
Sure, it can be an `Err` variant. But the user still needs a way to extract the receipts and logs from there!

Reply:
What I suggest is we merge as-is so we at least have logging, and then we open an issue to make it not an `Error`

Reply:
Okay. Well at least the current is better than what was available before.